PulsePlayer/PulsePlayer/library.json

```diff
@@ -3,94 +3,100 @@
   "content-title":     "No ads",
   "content-id":        "demo-none",
   "content-duration":  52,
-  "content-url":       "http://pulse-demo.cdn.videoplaza.tv/resources/media/sintel_trailer_854x480.mp4"
+  "content-url":       "http://pulse-demo.vp.videoplaza.tv/resources/media/sintel_trailer_854x480.mp4"
+},{
+    "tags":             ["om_vast4.1_wrapper","om_vast3.0"],
+  "midroll-positions": [10, 20],
+  "content-title":    "OMSDK",
+  "content-duration": 52,
+  "content-url":      "https://pulse-demo.vp.videoplaza.tv/resources/media/sintel_trailer_854x480.mp4"
```

Comment:
Would it make sense to use HTTPS content URLs in all these examples?

Reply:
sure, will change.